data/sql_updates/omnibus_dates.sql

```diff
@@ -64,7 +64,7 @@ with elections_raw as(
                 ], '-')
             else elections_raw.election_state
         end as report_contest
-    from disclosure.trc_report_due_date reports
+    from trc_report_due_date reports
```

Comment:
Did this change get rolled in due to switching the target branch from `develop` to `master`?  Will this be okay?